lib/modules/manager/pub/artifacts.spec.ts

```diff
@@ -0,0 +1,265 @@
+import { join } from 'upath';
```

Comment:
Please put the correct lockfile name and format for the `pub` manager in the list here:

https://github.com/renovatebot/renovate/blob/66159ac549e4ecb8e2425193b1d40bb368576bb0/docs/usage/configuration-options.md?plain=1#L1413-L1424